R/getPrevalence.R

```diff
@@ -204,13 +213,13 @@ setMethod("getPrevalence", signature = c(x = "ANY"),
             stop("'sort' must be TRUE or FALSE.", call. = FALSE)
         }
         #
-
         if (include_lowest) {
             prev <- x >= detection
         } else {
             prev <- x > detection
         }
-        prev <- rowSums(prev)
+        # Calculate prevalence for each taxa
+        prev <- rowSums(prev, na.rm = na.rm)
```

Comment:
I am still just wondering if we should throw a warning in a case where NAs actually exist & actively ignored? (ignoring NAs is now the default, user might not notice that this happens unless they are warned)

Reply:
That is good and adds transparency